data/transactions/logic/opcodes.go

```diff
@@ -798,6 +799,17 @@ var OpSpecs = []OpSpec{
 		costByField("g", &EcGroups, []int{
 			BN254g1: 630, BN254g2: 3_300,
 			BLS12_381g1: 1_950, BLS12_381g2: 8_150})},
+	{0xe6, "mimc", opMimc, proto("b:b{32}"), mimcVersion, costByFieldAndLength("c", &MimcConfigs, []linearCost{
+		BN254Mp110: {
+			baseCost:  10,
+			chunkCost: 350,
```

Comment:
I'm getting much larger chunk costs.  Here's a benchmark run:
```
BenchmarkHashes/mimc_BN254Mp110-32-10         	  146948	      8149 ns/op	         3.000 extra/op
BenchmarkHashes/mimc_BN254Mp110-128-10        	   36838	     32760 ns/op	         3.000 extra/op
BenchmarkHashes/mimc_BN254Mp110-512-10        	    8864	    133376 ns/op	         3.000 extra/op
BenchmarkHashes/mimc_BN254Mp110-1024-10       	    4384	    271565 ns/op	         3.000 extra/op
BenchmarkHashes/mimc_BN254Mp110-4096-10       	    1083	   1106159 ns/op	         3.000 extra/op
```
So that means the 12 chunk run (BN254Mp110-512-10) took 133376ns, and the 4 chunk run (BN254Mp110-128-10) took 32760ns. So 8 chunks extra took 100616ns.  Rough rule of thumb, 15ns per "tick", gives a perchunk cost of (100616/8)/15 = 838.

Reply:
I'll update the cost